src/app/components/form/NumberField.tsx

```diff
@@ -0,0 +1,100 @@
+import { useRef } from "react";
+import { classNames } from "~/app/utils";
+
+export type Props = {
+  suffix?: string;
+  endAdornment?: React.ReactNode;
+  secondaryValue: string | number;
+  label: string;
+};
+
+export default function DualCurrencyInput({
```

Comment:
All the imports from `NumberField` are called `DualCurrencyInput`, like this function. Should we align the file-name as well to make this clear?